docs/docs-components/GeneratedPropTable.js

```diff
@@ -104,7 +104,7 @@ export default function GeneratedPropTable({
         .replace(/ComponentType/g, 'React.ComponentType')
         // Replace "Element" with "React.Element" to match docs convention
         // Includes `<` to avoid picking up `HTMLDivElement` and similar
-        .replace(/^Element</g, 'React.Element<')
+        .replace(/Element</g, 'React.Element<')
```

Comment:
Huh. Not only was I wrong about possibly needing more targeted logic here, this is actually catching some cases I'd missed! 🎉 
![Screen Shot 2022-11-16 at 2 09 18 PM](https://user-images.githubusercontent.com/12059539/202305464-d04ae31c-d0e0-4a2b-a336-a69643948fce.png)
![Screen Shot 2022-11-16 at 2 10 32 PM](https://user-images.githubusercontent.com/12059539/202305468-7714dffb-350f-4f80-a0c5-9b4c32a34776.png)